app/views/homes/index.html.erb

```diff
@@ -1,4 +1,32 @@
-<p>
-  Modify me in `app/views/homes/index.html.erb` or change the root directive in
-  `config/routes.rb`
-</p>
+<div class="row">
+  <div class="large-12 columns">
+    <div class="row">
+      <div class="splash-title panel large-centered small-centered
+       large-4 medium-4 small-4 columns">
```

Comment:
You have `large`, `medium`, and `small` all set to the same things. That means you can JUST set `small-4` and by default, `medium` and `large` will decide to be the same as whatever `small` is.